lib/ansible/modules/windows/win_dns_client.ps1

```diff
@@ -193,29 +209,29 @@ Try {
 
     Write-DebugLog ("Validating adapter name {0}" -f $adapter_names)
 
-    $adapters = @($adapter_names)
-
-    If($adapter_names -eq "*") {
-        $adapters = Get-NetAdapter | Select-Object -ExpandProperty Name
-    }
     # TODO: add support for an actual list of adapter names
     # validate network adapter names
-    ElseIf(@(Get-NetAdapter | Where-Object Name -eq $adapter_names).Count -eq 0) {
-        throw "Invalid network adapter name: {0}" -f $adapter_names
+    $adapter_names = @($adapter_names)
+
+    foreach($adapter_name in $adapter_names) {
+        If($adapter_name -eq "*") {
+            $adapters = Get-NetAdapter | Select-Object -ExpandProperty Name
```

Comment:
I would write `(Get-NetAdapter).Name` but that's just me, performance wise it's the same

Reply:
Actually I missed this line, I just changed the indention... It is now fixed.